pycalphad/io/tdb.py

```diff
@@ -541,7 +541,7 @@ def reflow_text(text, linewidth=80):
         else:
             while len(line) > linewidth:
                 linebreak_idx = linewidth - 1
-                while linebreak_idx > 0 and line[linebreak_idx] not in linebreak_chars:
+                while linebreak_idx > 0 and not re.match(r'[^E\(][+-][0-9]', line[linebreak_idx-1:linebreak_idx+2]) and line[linebreak_idx] not in linebreak_chars:
```

Comment:
Can you clarify why this regex is needed?

Reply:
See response in https://github.com/pycalphad/pycalphad/pull/516#issuecomment-2186652180